src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -126,7 +127,8 @@ public static final String getHelpSummary() {
                 .append("Delete an inventory item: delete-i INDEX\n")
                 .append("Edit an inventory item: edit-i INDEX [n/NAME] [q/QUANTITY]"
                         + " [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]\n")
-                .append("Find an inventory item: find-i n/KEYWORD [MORE_KEYWORDS]\n")
+                .append("Find an inventory item: find-i [n/NAME | q/QUANTITY | s/SUPPLIER | "
+                        + "max/MAX_QUANTITY | t/TAG]\n")
```

Comment:
You cannot find quantity and max quantity

Reply:
@zeranium97 updated, thanks!